package.json

```diff
@@ -133,6 +133,7 @@
   "resolutions": {
     "bl": "^4.0.3",
     "acorn-globals/acorn": "^6.4.1",
+    "@babel/preset-env": "7.13.15",
```

Comment:
Can you please add something to the PR description about why are new dependency was needed? More for archive purposes if we ever need to remember in what context this was added its good to also have some description in the PR.